docs/source/create/entities/avatar-interactions.md

```diff
@@ -1,12 +1,20 @@
 # Define Interactions with Avatars
 
+Your avatar can interact with different types of entities. You can change the color of a cube entity as your avatar moves closer to it, or create an avatar entity that travels with you through different domains. 
```

Comment:
```suggestion
In real life, you interact with objects on a daily basis. In High Fidelity, your avatar can also interact with objects (entities) in the metaverse. There are a number of ways you can define the interactions you have with objects: you can write scripts to change the properties of an entity. You can create entities that are unique to your avatar (we call these "avatar entities"). And don't forget that you can set an entity's behavior and collision properties, so that objects are grabbable, triggerable, or dynamic.
```